setup.cfg

```diff
@@ -94,6 +94,8 @@ datasets =
     radiant-mlhub>=0.2.1,<0.5
     # rarfile 3+ required for correct Rar file detection
     rarfile>=3,<5
+    # scikit-image required for find_contours
+    scikit-image>=0.19.0,<1
```

Comment:
`find_contours` existed long before scikit-image 0.19, what's the minimum version that works? I try not to require newer versions unless absolutely necessary. The easiest way to test this would be to pip install successively older versions of scikit-image and see what the oldest working version is.

Reply:
`find_contours` was added in `v0.4`. 

I tried installing previous versions of `scikit-image` (while keeping the other dependencies constant) and they failed for one reason or the other. Downgrading numpy and scipy might allow those versions to work but I don't think that's the way to go.

For reference, I tested the versions via the following:
```
# Create test environment and install minimal dependencies
conda create -n min-test python=3.7
pip install -r requirements/min.old

# Install version of scikit-image 
pip install scikit-image=A.B.C

# Run test
pytest tests/datasets/test_vhr10.py

```

Reply:
I'll try to play around with this when I get a chance and see if I can get older versions working without any code changes.

Reply:
Tried this again. `0.19.0` is the oldest version that works consistently. `0.18.0` works on Python 3.7 but fails on others. Had to update scipy as well.